repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/atomfeed/FeedCacheInvalidator.java

```diff
@@ -77,15 +78,20 @@ public void setFeeds(List<AuthFeedReader> feeds) {
 
     @Override
     public void run() {
+
+        // Generate trans-id here so it is the same between multiple pages
+        String traceID = UUID.randomUUID().toString();
         while (!done) {
 
-            List<String> userKeys = new ArrayList<String>();
-            List<String> tokenKeys = new ArrayList<String>();
+            LOG.debug("Feed invalidation request for trans-id: " + traceID);
```

Comment:
you don't need to explicitly add the trace id, if you put it in the context it'll get there automatically.